dataeng/resources/token-helper.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
```

Comment:
By specifying this shebang, you are explicitly pointing it to the system python binary, not the one specified in the virtualenv. This may cause errors.